src/nsupdate/urls.py

```diff
@@ -2,8 +2,6 @@
 top-level url dispatching
 """
 
-import six
-
```

Comment:
btw, for getting CI working in current master branch, I had to add `six` to setup.py install_requires.

Reply:
removed again from this PR.